api/projects/projects.go

```diff
@@ -32,3 +36,47 @@ func create(c echo.Context) error {
 
 	return c.String(http.StatusOK, "Created project: "+projectName)
 }
+
+func list(c echo.Context) error {
+	db, ok := c.Get(constants.CTX_DB).(*gorm.DB)
+	if !ok {
+		logrus.WithFields(logrus.Fields{
+			"context": c,
+		}).Error("Unable to get DB from context in project create")
+		return errors.New("Unable to get DB from context")
+	}
+
+	projects := db.Find(&[]models.Project{})
+	return c.JSON(http.StatusOK, projects)
```

Comment:
please ensure that this returns an empty [] for no results, and not null. By default, json encoding a nil slice will return null. you actually need to make the list to get [], so please just make sure.